x-pack/plugins/security/server/routes/api/public/roles/get.js

```diff
@@ -15,13 +15,51 @@ export function initGetRolesApi(server, callWithRequest, routePreCheckLicenseFn,
     const roleKibanaApplications = roleApplications
```

Comment:
We already discussed this, but we'll need changes here to support the `kibana-*` wildcard entry on the reserved role definitions.